pipeline-modules/app-service/azure/sonarqube.yaml

```diff
@@ -7,66 +7,61 @@ description: Analyze source code using Sonar Qube
 target: ""
 category: code analysis
 keywords:
-  - Analyze Code
+  - analyze
+  - quality gate
 author: CloudCover
 meta: {}
+env:
+  - CPI__SONARQUBE_SERVICE_CONNECTION
 inputs:
   properties:
-    git_checkout_target:
-      title: Checkout Target
-      description: Git branch, tag or commit hash to checkout.
+    applicationName:
+      title: Application name
+      description: Codepipes application name
```

Comment:
Find anywhere that you have "codepipes" or "Codepipes" in a description and change to Code Pipes